graphics/src/widget/canvas.rs

```diff
@@ -28,7 +28,7 @@ pub use stroke::{LineCap, LineDash, LineJoin, Stroke};
 pub use text::Text;
 
 use crate::{Backend, Primitive, Renderer};
-pub use crate::gradient::Gradient;
+pub use crate::gradient::{Gradient, Position, Location};
```

Comment:
I'm not sure we should re-export `Position` and `Location` directly from `canvas` since they won't have the context of `gradient`.

```rs
use iced::widget::canvas::gradient;

gradient::Position {
    ..
}
```

vs 

```rs
use iced::widget::canvas::Position;

// Too generic, Position can be for so many things in a 2D canvas API
Position {
    ..
}
```